components/org.wso2.carbon.identity.sso.agent/src/main/java/org/wso2/carbon/identity/sso/agent/secretkey_temp.txt

```diff
@@ -0,0 +1 @@
+1234567812345678
```

Comment:
`src/main/` location should not be used to place resource files. This needs to go to `src/resources/META-INF/` folder.